admin_manual/maintenance/manual_upgrade.rst

```diff
@@ -21,6 +21,12 @@ First, :doc:`backup <backup>` the following items:
 
   cp -rv /var/www/owncloud /opt/backup/owncloud
   mysqldump <db_name> > /opt/backup/backup-file.sql
+  
+Check ownCloud's Mandatory Requirements
+---------------------------------------
+
+:ref:`ownCloud's mandatory requirements <prerequisites_label>` (such as PHP extensions) can change from version to version. 
+Ensure that you review them and update your servers to meet them, if one or more are missing. 
```

Comment:
...(such as PHP versions or PHP extensions)
...Ensure that you review them and update your server if required before upgrading ownCloud.